Source/Core/Core/AchievementManager.h

```diff
@@ -20,6 +20,8 @@
 #include "Common/WorkQueueThread.h"
 
 using AchievementId = u32;
+constexpr size_t FORMAT_SIZE = 24;
```

Comment:
Might want to make this a more unique name. This is pretty generic for something that's in the global namespace